test/python/dialects/indexing/dialect.py

```diff
@@ -91,7 +91,7 @@ def test_tensor_value():
   print(module)
 
   pm = PassManager.parse(
-      'builtin.module(func.func(convert-elementwise-to-linalg))')
+      "builtin.module(func.func(convert-elementwise-to-linalg))")
```

Comment:
Why this change?

Reply:
either yapf or black did this. i'll undo it